src/dsvpn.c

```diff
@@ -76,6 +76,13 @@ typedef struct Context_ {
     uint32_t      uc_st[2][12];
 } Context;
 
+int signal_toggle = 0;
```

Comment:
`signal_toggle` should be of `volatile sig_atomic_t` type.

Reply:
TIL